packages/core/src/keyBinding/keyBinding.ts

```diff
@@ -131,14 +146,26 @@ export class KeyBinding<
         >
     }
 
+    public withSdHash(sdHash: string) {
```

Comment:
Not a 100% whether it would be inline, but for this API I'd prefer the name `withSdHashClaim` as the other `withXXX` methods add a property to the class and not the payload.

Reply:
Makes sense 👍